### DISCUSSION TITLE
Base Url for photos

### DISCUSSION THREAD

**Discussion by TasosBak** [alexzaganelli/strapi-plugin-email-designer#76] [Ideas]
Hello guys, 

Is there a way to have a baseSrcUrl variable inside the template? The use case is that the project will run in multiple environments with different base urls (e.g. localhost:1337, another server's IP address etc). Another app will consume the email templates from strapi, populate the variables with the correct values with the help of a template engine and then send the email. The thing is that with that solution, I cannot preview the email template in strapi because the url is not yet populated.

Is there any way that I can make the previewer work?

`<img
                    align="center"
                    th:src="@{​​​​​​​​http://{​​​​​​​​baseSrcUrl}​​​​​​​​/uploads/photo_d9847b7078.png(baseSrcUrl = ${​​​​​​​​baseSrcUrl}​​​​​​​​)}​​​​​​​​"
                    alt="photo"
                    width="600"
                    height="200"
                />`

Thank you for your time guys!

**Comment by alexzaganelli** [comment]
Hi @TasosBak,  I'm currently abroad on holiday and I'm not able to help you but I'll try for sure as soon as I return home.
